openhtf/util/console_output.py

```diff
@@ -77,6 +92,8 @@ def banner_print(msg, color='', width=60, file=sys.stdout, logger=_LOG):
     ======================== Foo Bar Baz =======================
 
   """
+  if CLI_QUIET:
+    return
   logger.debug(ANSI_ESC_RE.sub('', msg))
```

Comment:
The behavior wrt logging seems inconsistent. In most places in this file, we continue to log messages if `CLI_QUIET` is False. Here, in `banner_print`, we don't log if `CLI_QUIET` is False. In `bracket_print` there is no logging at all.